test/e2e/tests/ngJqSpec.js

```diff
@@ -0,0 +1,12 @@
+describe('Customizing the jqlite / jquery version', function() {
+
+  it('should be able to force jqlite', function() {
+    loadFixture("ngJq").andWaitForAngular();
+    expect(browser.executeScript('return window.angular.element !== window.jQuery')).toBe(true);
+  });
+
+  it('should be able to use a specific version jQuery', function() {
+    loadFixture("ngJqJquery").andWaitForAngular();
+    expect(browser.executeScript('return window.angular.element !== window.jQuery_2_0_0')).toBe(true);
```

Comment:
The problem we have here is that e2e tests do not run in the context of the browser. In fact they do not run on the browser at all. Protractor converts JS in the tests into commands that are sent via Selenium to the browsers. We have to consider the browser a black box.

(OK, so it is possible to send a JavaScript block to be run on the browser but this is not the best approach).

Instead what we want is the fixture applications to expose the information about JQuery/jqLite in HTML that we can then inspect using Protractor